HdcpProfile/HdcpProfile.cpp

```diff
@@ -41,11 +43,13 @@ namespace WPEFramework
     namespace Plugin
     {
         SERVICE_REGISTRATION(HdcpProfile, 1, 0);
+        static const char* getHdcpReasonStr (int eHDCPEnabledStatus);
 
         HdcpProfile* HdcpProfile::_instance = nullptr;
 
         HdcpProfile::HdcpProfile()
         : AbstractPlugin()
+        , m_apiVersionNumber(1)
```

Comment:
We don't need m_apiVersionNumber. Versioning is handled in the base class (AbstractPlugin)